package.json

```diff
@@ -26,8 +26,10 @@
     "@babel/core": "^7.7.0",
     "@storybook/react": "^3.2.17",
     "@storybook/storybook-deployer": "^2.0.0",
-    "@theforeman/builder": "^4.2.0",
-    "@theforeman/vendor-dev": "^4.2.0",
+    "@testing-library/jest-dom": "^5.3.0",
+    "@testing-library/react": "^10.0.2",
```

Comment:
Opened up https://github.com/theforeman/foreman-js/pull/134 for react-testing-lib dep